tools/gulptasks/dashboards/api-server.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (C) Highsoft AS
+ */
+
+const gulp = require('gulp');
+const path = require('path');
+
+/* *
+ *
+ *  Constants
+ *
+ * */
+
+const MIMES = {
+    css: 'text/css',
+    eot: 'application/vnd.ms-fontobject',
+    js: 'application/javascript',
+    json: 'application/json',
+    html: 'text/html',
+    ico: 'image/x-icon',
+    png: 'image/png',
+    svg: 'image/svg+xml',
+    ttf: 'font/ttf',
+    txt: 'text/plain',
+    woff: 'font/woff',
+    woff2: 'font/woff2',
+    xml: 'application/xml'
+};
+
+const SOURCE_PATH = path.join(__dirname, '..', '..', '..', 'build', 'api');
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+
+/**
+ * Response with a 200
+ *
+ * @param {ServerResponse} response
+ * HTTP response
+ *
+ * @param {Buffer} data
+ * File data
+ *
+ * @param {string} ext
+ * File extension
+ *
+ * @return {void}
+ */
+function response200(response, data, ext) {
+    response.writeHead(200, { 'Content-Type': MIMES[ext] || MIMES.html });
+    response.end(data);
+}
+
+/**
+ * Response with a 302 - redirect
+ *
+ * @param {ServerResponse} response
+ * HTTP response
+ *
+ * @param {string} p
+ * Redirect path
+ *
+ * @return {void}
+ */
+function response302(response, p) {
+    response.writeHead(302, { Location: p });
+    response.end();
+}
+
+/**
+ * Response with a 404 - not found
+ *
+ * @param {ServerResponse} response
+ * HTTP response
+ *
+ * @param {string} uri
+ * Missing URL path
+ *
+ * @return {void}
+ */
+function response404(response, uri) {
+
+    const logLib = require('../lib/log');
+
+    logLib.failure('404', uri);
+
+    response.writeHead(404);
+    response.end('Ooops, the requested file is 404', 'utf-8');
+}
+
+/* *
+ *
+ *  Tasks
+ *
+ * */
+
+/**
+ * Start a server serving up the API documentation
+ *
+ * @return {Promise<void>}
+ *         Promise to keep
+ */
+function apiServer() {
+
+    const fs = require('fs');
+    const http = require('http');
+    const logLib = require('../lib/log');
+
+    return new Promise(resolve => {
+
+        const port = 9005;
+
+        http
+            .createServer((request, response) => {
+
+                let uri = request.url;
+
+                if (uri === '/dashboards' || uri === '/' || uri === '') {
+                    response302(response, '/dashboards/');
+                    return;
+                }
+                if (uri === '/highcharts') {
+                    response302(response, '/highcharts/');
+                    return;
+                }
+                if (uri === '/highstock') {
+                    response302(response, '/highstock/');
+                    return;
+                }
+                if (uri === '/highmaps') {
+                    response302(response, '/highmaps/');
+                    return;
+                }
+                if (uri === '/gantt') {
+                    response302(response, '/gantt/');
+                    return;
+                }
```

Comment:
How about a `switch`, or use `.match` and return a result there?

Just an idea.